### DISCUSSION TITLE
python using anaconda jupyter notebook

### DISCUSSION THREAD

**Discussion by fernandotoscano** [frontendbr/forum#2444] [Perguntas e Respostas]
I´m using anaconda's jupyter notebook and I got an error like this:

____________________________________________________________________________________________________________________________
____________________________________________________________________________________________________________________________
[/usr/local/lib/python3.10/dist-packages/pandas/core/series.py](https://localhost:8080/#) in __array__(self, dtype)
    891         array(['1999-12-31T23:00:00.000000000', ...],
    892               dtype='datetime64[ns]')"""
--> 893         return np.asarray(self._values, dtype)
    894 
    895     # ----------------------------------------------------------------------

ValueError: could not convert string to float: 'M'
________________________________________________________________________________________________________________________________
________________________________________________________________________________________________________________________________
I´m trying to learn in a Project named: Breast Cancer Detection Using Python & Machine Learning (youtube)

And when I try to run the code :  
# Visualize the count
sns.countplot(df['diagnosis'], label= 'count')

I received the error message above.

In the column 'diagnoses' I only have "M" for malignant and "B" benign.

Can some one help me please.
Thank You

**Comment by xiazhengtao** [comment]
what is this
